CWE/inventions/culture_inventions.txt

```diff
@@ -629,117 +666,123 @@ social_anthropology = {
-                base = 2
-                modifier = {
-                        factor = 12
-                        OR = { 
+	limit = { cold_war = 1 }
+	news = no
+	chance = {
+		base = 2
+		modifier = {
+			factor = 12
+			OR = { 
 				tag = USA 
 				tag = RUS 
 			} 
-                }
-                modifier = {
-                        factor = 2
-                        is_greater_power = yes
-                }
-        }
-        effect = {
-            permanent_prestige = 5
-            rebel_org_gain = {
-                faction = separatist_rebels
-                value = 0.50
-            }
-        }
+		}
+		modifier = {
+			factor = 2
+			is_greater_power = yes
+		}
+	}
+	effect = {
+		permanent_prestige = 5
+		rebel_org_gain = {
+		faction = separatist_rebels
+		value = 0.50
+		}
+	}
 }
 man_in_orbit = {
-        limit = { cold_war = 1 }
-        chance = {
-                base = 2
-                modifier = {
-                    factor = 12
-					OR = { 
-						tag = USA 
-						tag = RUS 
-					} 
-                }
-                modifier = {
-                        factor = 2
-                        is_greater_power = yes
-                }
-        }
-        effect = {
-            permanent_prestige = 2
-			education_efficiency = 0.1
-        }
+	limit = { cold_war = 1 }
+	news = no
+	chance = {
+		base = 2
+		modifier = {
+			factor = 12
+			OR = { 
+				tag = USA 
+				tag = RUS 
+			} 
+		}
+		modifier = {
+			factor = 2
+			is_greater_power = yes
+		}
+	}
+	effect = {
+		permanent_prestige = 2
+		education_efficiency = 0.1
+	}
 }
 spacewalk = {
-        limit = { cold_war = 1 }
-        chance = {
-                base = 2
-                modifier = {
-                    factor = 12
-					OR = { 
-						tag = USA 
-						tag = RUS 
-					} 
-                }
-                modifier = {
-                        factor = 2
-                        is_greater_power = yes
-                }
-        }
-        effect = {
-            permanent_prestige = 2
-			education_efficiency = 0.1
-        }
+	limit = { cold_war = 1 }
+	news = no
+	chance = {
+		base = 2
+		modifier = {
+			factor = 12
+			OR = { 
+				tag = USA 
+				tag = RUS 
+			} 
+		}
+		modifier = {
+			factor = 2
+			is_greater_power = yes
+		}
+	}
+	effect = {
+		permanent_prestige = 2
+		education_efficiency = 0.1
+	}
 }
 space_docking = {
-        limit = { detente = 1 }
-        chance = {
-                base = 2
-                modifier = {
-                    factor = 12
-					OR = { 
-						tag = USA 
-						tag = RUS 
-					} 
-                }
-                modifier = {
-                        factor = 2
-                        is_greater_power = yes
-                }
-        }
-        effect = {
-            permanent_prestige = 2
-			education_efficiency = 0.1
-        }
+	limit = { detente = 1 }
+	news = no
+	chance = {
+		base = 2
+		modifier = {
+			factor = 12
+			OR = { 
+				tag = USA 
+				tag = RUS 
+			} 
```

Comment:
Should check for space agency country modifier instead of tags to avoid rail roading